training/train.py

```diff
@@ -254,7 +243,7 @@ def save_model(s3_client, recommender):
         _logger.error(str(exc))
 
 
-def save_dictionaries(s3_client, package_id_dict, manifest_id_dict):
+def save_dictionaries(s3_client, package_id_dict, manifest_id_dict):  # pragma: no cover
```

Comment:
Yeah, there is no reason why save_model and save_dictionaries are two different units, like in this model can one even happen without the other?

Reply:
Well, I think no kind of saving can happen independently. I mean saving model, dictionaries and hyper parameters has to happen together in the current structure of the code. I think we can take this as a part of the refactoring.